packages/build-utils/test/unit.convert-runtime-to-plugin.test.ts

```diff
@@ -80,14 +119,54 @@ describe('convert-runtime-to-plugin', () => {
       },
     });
 
+    const routesManifest = JSON.parse(output['routes-manifest.json']);
+    expect(routesManifest).toMatchObject({
+      version: 3,
+      pages404: true,
+      redirects: [],
+      rewrites: [
+        /* TODO: `handle: miss`
+        {
+          source: "^/api/(.+)(?:\\.(?:py))$",
+          destination: "/api/db/[id]?id=$1",
+          regex: "^/api/(.+)(?:\\.(?:py))$"
+        }
+        */
+      ],
+      dynamicRoutes: [
+        {
+          page: '/api/project/[aid]/[bid]/index',
```

Comment:
@AndyBitz Does the page need to strip `/index` here?

Reply:
Good question. I think it doesn't matter, because `/index` and `/` should be equal

Reply:
Ah, I think it'd be best to remove it, as I just checked what Next.js does:

```
  "dynamicRoutes": [
    {
      "page": "/[nice]",
      "regex": "^/([^/]+?)(?:/)?$",
      "routeKeys": {
        "nice": "nice"
      },
      "namedRegex": "^/(?<nice>[^/]+?)(?:/)?$"
    }
  ]
```

for a `pages/[nice]/index.js`

Reply:
Not relevant because Next.js will not create an `index` file for it, it'll make `[nice]` the file itself, so yea, we can leave it as it is.